src/test/suite/fixtures/recorded/languages/java/changeCallee2.yml

```diff
@@ -0,0 +1,23 @@
+languageId: java
+command:
+  version: 1
+  spokenForm: change callee
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: functionCallee, includeSiblings: false}
+initialState:
+  documentContents: Main obj = new Main();
+  selections:
+    - anchor: {line: 0, character: 20}
+      active: {line: 0, character: 20}
+  marks: {}
+finalState:
+  documentContents: Main obj = new ();
```

Comment:
I'm wondering if I would not expect `new` to be part of the callee

Reply:
I think we need to make a decision on this because selecting the field as @pokey suggested does not select the new keyword in multiple languages. It does select everything after the 'new ' in cases like `new object.innerClass()`